src/guiguts/tools/pptxt.py

```diff
@@ -1271,32 +1205,32 @@ def unicode_numeric_character_check() -> None:
     )
 
     # Courtesy limit if user uploads fpgen source, etc.
+    courtesy_limit = 5
     numeric_char_reference_count = 0
+    lines_with_numeric_char_references = 0
 
     line_number = 1
     regx = r"(&#[0-9]{1,4};|&#x[0-9a-fA-F]{1,4};)"
```

Comment:
Shouldn't the decimal digits be `{1,5}`? `9999` decimal is considerably less than `ffff` hexadecimal (which I think covers the BMP?). (Though maybe I'm misunderstanding what's intended here.)

Reply:
It should be and I will fix that as well. Thanks for pointing it out.